modules/react/dialog/stories/examples/Basic.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import {Dialog} from '@workday/canvas-kit-react/dialog';
+import {PrimaryButton} from '@workday/canvas-kit-react/button';
+import {Flex} from '@workday/canvas-kit-react/layout';
+import {Basic as Input} from '../../../text-input/stories/examples/Basic';
```

Comment:
This `import {Basic as Input} from '../../../text-input/stories/examples/Basic'` prevents this example from being self-contained and copy-and-pasteable (it won't work on the Canvas Site either). You'll need to import TextInput and FormField from `@workday/canvas-kit-react` and include them in the example rather than importing the entire Basic Text Input example itself.